starcraft/__init__.py

```diff
@@ -14,10 +14,17 @@
 # You should have received a copy of the GNU General Public License along
 # with this program.  If not, see <http://www.gnu.org/licenses/>.
 """Starcraft package demo."""
-from importlib.metadata import version
 from typing import List, Optional, Any
 
-__version__ = version(__name__)
+try:
+    from ._version import __version__
+except ImportError:  # pragma: no cover
+    from importlib.metadata import version, PackageNotFoundError
+
+    try:
+        __version__ = version("starcraft")
+    except PackageNotFoundError:
+        __version__ = "dev"
 
 
 def hello(people: Optional[List[Any]] = None) -> None:
```

Comment:
I added this hello back because the tests (most importantly, the integration one) use it. If you have a better idea for this one I'm all ears